.travis.yml

```diff
@@ -1,23 +1,24 @@
-sudo: false
 language: php
+
+php:
+    - 5.4
+    - 5.5
+    - 5.6
```

Comment:
We should remove 5.4 and 5.5 because they are _dead_.

Reply:
I will remove these dead PHP versions.